tap_logmeinrescue/schemas/transferred_sessions_extended_report.json

```diff
@@ -11,7 +11,10 @@
     "waiting_time": {
       "type": ["null", "string"]
     },
-    "transferred_by_-_technician_id": {
+    "name": {
+      "type": ["null", "string"]
+    },
```

Comment:
my understanding is that `name` can be named something else. this is the purpose of the whole `header_to_string_schema` component in the tap -- it should add `name` to the schema there